app/views/directives/add-config-to-application.html

```diff
@@ -5,10 +5,14 @@
   <div class="dialog-body">
     <form name="addToApplicationForm" novalidate>
       <fieldset ng-disabled="disableInputs">
+        <div class="alert alert-warning" ng-if="ctrl.addType === 'env' && ctrl.application && !ctrl.canAddRefToApplication">
+          <span class="pficon pficon-warning-triangle-o" aria-hidden="true"></span>
+          <strong>Containers of the selected application already contain this {{ctrl.apiObject.kind | humanizeKind}}.</strong>
```

Comment:
Maybe just

```html
<strong>The {{ctrl.apiObject.kind | humanizeKind}} has already been added to this application.</strong>
```

Reply:
Suggest using the help-block / has-error style (instead of an alert) and moving it below the select.